source/background.js

```diff
@@ -5,9 +5,16 @@ import dynamicContentScripts from 'webext-dynamic-content-scripts';
 // Define defaults
 new OptionsSync().define({
 	defaults: {
-		hideStarsOwnRepos: true
+		disabledFeatures: 'log-active-features',
+		logFeatures: false
```

Comment:
I was thinking we make this more generic so we could use it for other production logging in the future. This locks it into just being logging of features.

Reply:
Done. When that happens the feature can be developed further.